src/components/Card.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import styled from 'styled-components'
+import twitterIcon from '../images/twitter_icon.png'
+import InstagramIcon from '../images/instagram_icon.png'
+import youtubeIcon from '../images/youtube_icon.png'
+import arrowDown from '../images/arrow-down.png'
+import bookmarkEmpty from '../images/Bookmark_empty.svg'
+import bookmarkFilled from '../images/Bookmark_Filled.png'
+
+const Wrapper = styled.section`
+  position: relative;
+  width: 80%;
+  min-height: 150px;
+  background: #fff;
+  box-shadow: 0 8px 16px rgba(0, 40, 100, 0.4);
+  border-radius: 4px;
+
+  img {
+    width: 100%;
+    height: 100px;
+    object-fit: cover;
+    border-radius: 4px 4px 0 0;
+  }
+`
+const CampaignInfo = styled.div`
+  position: relative;
+  display: grid;
+  grid-template-columns: auto auto;
+  padding: 0 15px 15px;
```

Comment:
Can you condense this into two values instead of three?

Reply:
I can't. Because Top and Bottom have different values